jsk_arc2017_baxter/euslisp/lib/arc-interface.l

```diff
@@ -344,42 +366,21 @@
       (setq obj-box-z-length (z-of-cube (send self :bbox->cube obj-box)))
       (setq obj-pos (v+ obj-pos (float-vector 0 0 (/ obj-box-z-length 2))))
       obj-pos))
-  (:try-to-pick-object
+  (:try-to-pick-object-v4
```

Comment:
currently `:larm` only have `:suction` grasping, so
API should be 
`(send *ri* :try-to-pick-object :larm :grasp-style :suction)` 
`(send *ri* :try-to-pick-object :rarm :grasp-style :suction)`
`(send *ri* :try-to-pick-object :rarm :grasp-style :pinch)`